extensions/amp-render/1.0/amp-render.js

```diff
@@ -41,7 +41,63 @@ const isAmpStateSrc = (src) => src && src.startsWith(AMP_STATE_URI_SCHEME);
  */
 const isAmpScriptSrc = (src) => src && src.startsWith(AMP_SCRIPT_URI_SCHEME);
 
-class AmpRender extends BaseElement {
+/**
+ * Gets the json from an "amp-state:" uri. For example, src="amp-state:json.path".
+ *
+ * TODO: this implementation is identical to one in amp-list. Move it
+ * to a common file and import it.
+ *
+ * @param {!AmpElement} element
+ * @param {string} src
+ * @return {Promise<!JsonObject>}
+ */
+const getAmpStateJson = (element, src) => {
```

Comment:
super nit: when making top level functions, I personally have a slight preference towards `function` vs. arrow funcs. That way definition order doesn't matter